.github/workflows/publish-and-deploy.yaml

```diff
@@ -31,12 +31,12 @@ jobs:
           context: .
           file: ./Dockerfile
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: willnilges/meshforms:helm
```

Comment:
What is the purpose of this?

Reply:
Make it not overwrite the `main` tag. The domain is hardcoded, so if it was pulled on grand it would break. Undo later or accept risk and undo now?

Reply:
Well, it seems like we would have to build two images: One for `prod` and one for `dev`. This won't keep both up to date with main, will it?

Reply:
Have a look